.circleci/config.yml

```diff
@@ -2,154 +2,16 @@ version: 2.1
-
-    - run:
-        name: Publish chart to CNR using a temporary channel for testing
-        command: ./architect publish --pipeline=false --channels=${CIRCLE_SHA1}-${E2E_TEST_NAME}
-
-    - run:
-        name: Run test
-        command: |
-          docker run --rm --network host \
-            -v $(pwd)/${E2E_TEST_DIR}:/e2e \
-            -v $(pwd):/go/src/github.com/${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME} \
-            -e E2E_KUBECONFIG=/e2e/kubeconfig \
-            -e CIRCLE_SHA1=${CIRCLE_SHA1} \
-            -w /go/src/github.com/${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME} \
-            golang:1 go test -v -tags k8srequired ./${E2E_TEST_DIR}/
-
-    - run:
-        name: Export logs
-        command: |
-          ./e2ectl logs export --logs-dir=kind/logs
-        when: always
-
-    - store_artifacts:
-        path: kind/logs
-        destination: logs
-
-    - run:
-        name: Finish with cleanup, no matter if the test succeeded or not
-        command: |
-          ./e2ectl cluster delete
-          ./architect unpublish --channels=${CIRCLE_SHA1}-${E2E_TEST_NAME}
-        when: always
-
-jobs:
-  build:
-    machine: true
-    steps:
-    - checkout
-
-    - run:
-        name: Get architect binary
-        command: |
-          wget -q $(curl -sS -H "Authorization: token $RELEASE_TOKEN" https://api.github.com/repos/giantswarm/architect/releases/latest | grep browser_download_url | head -n 1 | cut -d '"' -f 4)
-          chmod +x ./architect
-          ./architect version
-
-    - run: ./architect build
-
-    - run:
-        name: Publish chart to CNR using a temporary channel for feature branch deploys
-        command: ./architect publish --pipeline=false --channels=wip-${CIRCLE_SHA1}
-
-    - persist_to_workspace:
-        root: .
-        paths:
-        - ./architect
-
-  e2eTestBasic:
-    environment:
-      E2E_TEST_DIR: "integration/test/basic"
-      E2E_TEST_NAME: "basic"
-    <<: *e2eTest
-
-  e2eTestMetrics:
-    environment:
-      E2E_TEST_DIR: "integration/test/metrics"
-      E2E_TEST_NAME: "metrics"
-    <<: *e2eTest
-
-  deploy:
-    machine: true
-    steps:
-    - checkout
-
-    - attach_workspace:
-        at: .
-
-    - run: ./architect deploy
-
-    - run:
-        name: Publish chart to beta channel on CNR
-        command: ./architect publish
-
-  publish_to_stable:
-    machine: true
-    steps:
-    - checkout
-
-    - attach_workspace:
-        at: .
-
-    - run: ./architect publish --stable
-
 workflows:
   version: 2
-  build_e2e:
+  package-and-push-chart-on-tag:
     jobs:
-      - build
-
       - architect/push-to-app-catalog:
-          name: "package and push kubernetes-metrics-server chart"
+          name: "package and push metrics-server-app chart"
           app_catalog: "giantswarm-catalog"
           app_catalog_test: "giantswarm-test-catalog"
-          chart: "kubernetes-metrics-server-chart"
+          chart: "metrics-server-app"
```

Comment:
Sorry we need to use `default-catalog` and `default-test-catalog` here. See https://github.com/giantswarm/chart-operator/pull/286#issue-317932168.

Reply:
I already follow as you did, Thanks anyway.